ESPWebDAV.cpp

```diff
@@ -810,6 +810,7 @@ void ESPWebDAVCore::sendPropResponse(bool isDir, const String& fullResPathFS, si
 {
     String fullResPath = fullResPathFS;
     replaceFront(fullResPath, _fsRoot, _davRoot);
+    fullResPath = c2enc(fullResPathFS);
```

Comment:
`replaceFront()`'s result is overwritten.
Would you mind doing this instead ?
```cpp
    String blah;
    blah.reserve(256);
    blah = fullResPathFS;
    replaceFront(blah, _fsRoot, _davRoot);
    String fullResPath = c2enc(blah);
    blah.clear();
(... blah += F("<D:response)
```

I reckon this is not very "fluent" code, maybe c2enc needs a relifting to avoid more copies.

Reply:
What ever works.  :)